onnxruntime/core/providers/cuda/cuda_execution_provider.h

```diff
@@ -177,9 +177,12 @@ class CUDAExecutionProvider : public IExecutionProvider {
     }
 
     bool IsGraphCaptureAllowed() const;
+    bool IsGraphCaptureAllowedOnRun() const;
     void CaptureBegin();
     void CaptureEnd();
     bool IsGraphCaptured() const;
+    Status SetCudaGraphAnnotationId(const onnxruntime::RunOptions& run_options);
+    Status SetCudaGraphAnnotationId(CudaGraphAnnotation_t cuda_graph_annotation_id);
```

Comment:
It is not needed (See other comments), may change this to GetCudaGraphAnnotationId()